packages/sanes-chrome-extension/package.json

```diff
@@ -14,7 +14,7 @@
     "build-staging": "yarn use-config-staging && yarn build-with-any-config && yarn make-manifest-staging",
     "build-production": "yarn use-config-production && yarn build-with-any-config && yarn make-manifest-production",
     "build": "yarn build-development",
-    "override-config-development": "cp ./config/development.json ./build/assets/config/conf.json && yarn make-manifest-development",
+    "override-config-development": "mkdir -p ./build/assets/config && cp ./config/development.json ./build/assets/config/conf.json && yarn make-manifest-development",
```

Comment:
Why this change? `override-config-development` is used to override the config of an existing build directory. When this is called before `yarn build`, this is a bug in the code that calls it.

Reply:
You are right I need to use this `override` only during e2e test, so this code will go away. I added this because `bierzo-wallet` was built before `extension`. Will fix now.